frontend/src/components/SocialMedia.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <div id="media">
+        <h2>MEDIA</h2>
+        <ul>
+            <li v-for="item in media" :key="item.link">
+                <a :href="item.link" target="_blank" class="circle">
+                    <v-icon> {{item.icon}} </v-icon>
+                </a>
+            </li>
+        </ul>
+    </div>
+</template>
```

Comment:
Please look up the vue component structure proposed in #73. The main idea behind it is to avoid scrolling between logic and template which can get a real pain in the bottom when the .vue file gets longer. Accordingly template should be separate .html file imported in .vue file.

Reply:
I know the idea, but Adam said that for now we should stick to old project structure (html, js and scss in one file).